File: server/当前版本/server/zlj/game_server/src/com/imop/lj/gameserver/corps/msg/CGOpenCorpsMemberList.java
package com.imop.lj.gameserver.corps.msg;

import com.imop.lj.core.msg.MessageType;
import com.imop.lj.gameserver.common.msg.CGMessage;
import com.imop.lj.gameserver.corps.handler.CorpsHandlerFactory;

/**
 * 打开军团成员列表
 * 
 * @author CodeGenerator, don't modify this file please.
 */
public class CGOpenCorpsMemberList extends CGMessage{
	
	
	public CGOpenCorpsMemberList (){
	}
	
	
	@Override
	protected boolean readImpl() {


		return true;
	}
	
	@Override
	protected boolean writeImpl() {

		return true;
	}
	
	@Override
	public short getType() {
		return MessageType.CG_OPEN_CORPS_MEMBER_LIST;
	}
	
	@Override
	public String getTypeName() {
		return "CG_OPEN_CORPS_MEMBER_LIST";
	}


	@Override
	public void execute() {
		CorpsHandlerFactory.getHandler().handleOpenCorpsMemberList(this.getSession().getPlayer(), this);
	}
}

File: u3u2/Assets/Scripts/app/net/c2s/CGOpenCorpsMemberList.cs
using System;
using System.IO;
namespace app.net
{

/**
 * 打开军团成员列表
 * 
 * @author CodeGenerator, don't modify this file please.
 */
public class CGOpenCorpsMemberList :BaseMessage
{
	
	
	public CGOpenCorpsMemberList ()
	{
	}
	
	
	protected override void ReadImpl() 
	{
		return;
	}
	
	protected override void WriteImpl() 
	{

	}
	
	public override short GetMessageType()
	{
		return (short)MessageType.CG_OPEN_CORPS_MEMBER_LIST;
	}
	
	public override string getEventType()
	{
		return "";
	}
	
	}
}